pre_commit/xargs.py

```diff
@@ -168,9 +172,49 @@ def xargs(
     def run_cmd_partition(
             run_cmd: tuple[str, ...],
     ) -> tuple[int, bytes, bytes | None]:
-        return cmd_fn(
-            *run_cmd, check=False, stderr=subprocess.STDOUT, **kwargs,
+        if not stream_output:
+            return cmd_fn(
+                *run_cmd, check=False, stderr=subprocess.STDOUT, **kwargs,
```

Comment:
I'm a little surprised by `stderr=subprocess.STDOUT`. Does that mean precommit streams stderr of the process to stdout already?

Reply:
i feel like i should have noticed some output, since clyde logs to stderr

Reply:
Probably depends on what functions these cmd_fn are?